thunder/tests/opinfos.py

```diff
@@ -1176,6 +1176,27 @@ def _abs_torch(x: torch.Tensor | Number):
 )
 elementwise_unary_ops.append(floor_opinfo)
 
+frexp_opinfo = OpInfo(
+    clang.frexp,
+    supports_grad=True,
+    dtypes=(datatypes.floating,),
+    sample_input_generator=partial(elementwise_unary_generator, small=True),
```

Comment:
Could you make an issue about vjp tests failing for `frexp` for large samples and mention it in a TODO comment here?

Reply:
done! Issue: #1991

Reply:
Thank you!